ui/lib/pki/addon/components/page/pki-configuration-edit.hbs

```diff
@@ -0,0 +1,107 @@
+            {{! boolean value is "disable" so false means enabled and ttl is toggled on}}
+            <div class="field">
+              <TtlPicker
+                data-test-input={{attr.name}}
+                @onChange={{fn this.handleTtl attr}}
+                @label={{if (get @crl attr.options.mapToBoolean) attr.options.labelDisabled attr.options.label}}
+                @helperTextDisabled={{attr.options.helperTextDisabled}}
+                @helperTextEnabled={{attr.options.helperTextEnabled}}
+                @initialValue={{get @crl attr.name}}
+                @initialEnabled={{not (get @crl attr.options.mapToBoolean)}}
+              />
+            </div>
+          {{/if}}
+          {{#if
+            (and (eq @crl.disable false) (or (eq attr.name "autoRebuildGracePeriod") (eq attr.name "deltaRebuildInterval")))
+          }}
+            <div class="field">
+              <TtlPicker
+                data-test-input={{attr.name}}
+                @onChange={{fn this.handleTtl attr}}
+                @label={{if (get @crl attr.options.mapToBoolean) attr.options.label attr.options.labelDisabled}}
+                @helperTextDisabled={{attr.options.helperTextDisabled}}
+                @helperTextEnabled={{attr.options.helperTextEnabled}}
+                @initialValue={{get @crl attr.name}}
+                @initialEnabled={{get @crl attr.options.mapToBoolean}}
+              />
+            </div>
+          {{/if}}
+          {{#if (eq attr.name "ocspExpiry")}}
+            <h2 class="title is-4 has-bottom-margin-xs has-border-bottom-light has-bottom-padding-s">
+              Online Certificate Status Protocol (OCSP)
+            </h2>
+            {{! boolean value is "ocspDisable" so false means enabled and ttl is toggled on}}
+            <div class="field">
+              <TtlPicker
+                data-test-input={{attr.name}}
+                @onChange={{fn this.handleTtl attr}}
+                @label={{if (get @crl attr.options.mapToBoolean) attr.options.labelDisabled attr.options.label}}
+                @helperTextDisabled={{attr.options.helperTextDisabled}}
+                @helperTextEnabled={{attr.options.helperTextEnabled}}
+                @initialValue={{get @crl attr.name}}
+                @initialEnabled={{not (get @crl attr.options.mapToBoolean)}}
+              />
+            </div>
+          {{/if}}
+        {{/each}}
+      {{else}}
+        <EmptyState @title="You do not have permission to set the CRL." />
+      {{/if}}
+    </fieldset>
+
+    <div class="field is-grouped box is-fullwidth is-bottomless">
+      <div class="control">
+        <button type="submit" class="button is-primary" data-test-configuration-edit-save>
+          Save
+        </button>
+        <button
+          {{on "click" this.cancel}}
+          type="button"
+          class="button has-left-margin-s"
+          data-test-configuration-edit-cancel
+        >
+          Cancel
+        </button>
+      </div>
```

Comment:
This probably isn't necessary since you're transitioning to the configuration page on submit, but do you want to show disabled/loading states?

Reply:
good idea!